lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -422,7 +422,7 @@ def _section_course_info(course, access):
 
     if settings.ANALYTICS_DASHBOARD_URL:
         dashboard_link = _get_dashboard_link(course_key)
-        message = _("Enrollment data is now available in {dashboard_link}.").format(dashboard_link=dashboard_link)
+        message = Text(_("Enrollment data is now available in {dashboard_link}.")).format(dashboard_link=dashboard_link)
```

Comment:
`_get_dashboard_link` brings back safe HTML. Previously, this was making this safe twice, which was spitting out HTML rather than the rendered HTML. Using `Text()` here ensures it uses the already safe HTML it receives from the `_get_dashboard_link` method.